packages/types/src/promotion/common/promotion-rule.ts

```diff
@@ -0,0 +1,23 @@
+import { BaseFilterable } from "../../dal"
+
+export type PromotionRuleOperator = "gt" | "lt" | "eq" | "ne" | "in"
```

Comment:
suggestion: Make common

Reply:
This is the type of the values of the `enum` PromotionRuleOperator. Should I just change the name to `PromotionRuleOperatorValues`?